games/sdl_sopwith/Portfile

```diff
@@ -23,10 +23,16 @@ checksums           md5     9527c13fd463731d0e189deb24eb39f8 \
                     sha256  afef3efb49994aec6a37185bde97af46d7231a7bf18dfbc6d3c87251f3f9dd46 \
                     size    1046380
 
-depends_build       port:pkgconfig
+depends_build       port:pkgconfig \
+                    port:automake
 depends_lib         port:libsdl2
 
-configure.args      --disable-sdltest
+patchfiles          src_sdl_Makefile.in.patch
+
+post-patch {
+    file delete ${worksrcpath}/config.cache ${worksrcpath}/config.log \
+                ${worksrcpath}/config.status
```

Comment:
It doesn't look like these exist in the current source distribution.

Reply:
Thanks, have fixed these. Automake added in error when .am was targeted for patching rather than .in; put back the post-patch stuff thinking it was needed to trigger rebuild from .am, but as you point out, these files are gone.